content/en/docs/started/workstation/getting-started-macos.md

```diff
@@ -4,44 +4,37 @@ description = "Install Kubeflow on macOS"
 weight = 20
                     
 +++
-{{% alert title="Out of date" color="warning" %}}
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
 
 For macOS systems you have multiple options for getting started. The options range
 from fully-assembled Kubeflow stacks, to stacks that require some assembly.
 
-## Kubeflow appliance
+*Note:* It is recommended that you have at least 12GB of RAM and 50GB of storage available.
 
-A Kubeflow appliance is a virtual machine that has Kubeflow already installed. Once the
-necessary supporting software is installed no further installation steps are required.
+### MicroK8s
 
-### MiniKF
+MicroK8s is a lightweight zero-ops Kubernetes which has a native installer for macOS. MicroK8s is highly available from 3+ nodes and includes a single-command install of Kubeflow.
 
-MiniKF is a predefined virtual machine that installs onto VirtualBox through Vagrant.
-The only following applications are required to use MiniKF:
+To get Kubeflow:
 
-- Install [Vagrant](https://www.vagrantup.com/downloads.html)
-- Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
+1. Install [MicroK8s](https://microk8s.io/) - see the "Alternative Install" page for links to the latest macOS installer.
```

Comment:
`see the "Alternative Install" page for links to the latest macOS installer.`
 
This does not seem to be needed, as the installer for macOS is in the main MicroK8s page.